base.php

```diff
@@ -2690,9 +2690,9 @@ protected function sandbox(array $hive=NULL,$mime=NULL) {
 			$hive=$fw->hive();
 		}
 		if ($this->level<1 || $implicit) {
-			if (!$fw->CLI && !headers_sent() &&
+			if (!$fw->CLI && $mime!=NULL && !headers_sent() &&
```

Comment:
Rest of the code follows in the minimalist style anyway so your comment is valid, I just liked to show it explicit because it has a strong functional effect.